contribute/style-guides/backend.md

```diff
@@ -75,3 +75,17 @@ Valid reasons to use a pointer include (but not necessarily limited to):
   allocating heap memory)
 * You might *need* `nil` to tell if a variable isn't set, although usually it's better to use the type's zero
   value to tell instead
+
+## Database
+
+In database related code, we follow certain patterns.
+
+### Foreign keys
+
+While they can be useful, we don't generally use foreign key constraints in Grafana, for historical and
```

Comment:
I will add explicit rationale next time:

I rephrased this so that if you took the content between commas out, the sentence still makes sense.
Also, "See this" should really be "Refer to" for consistency across docs.
Otherwise, we have "See", "Check out", "Read", "Refer to" etc to mean the same thing."

Reply:
@osg-grafana I also think @aknuds1 missed that there was a suggestion (and not just a quote) in addition to your question. If you use the suggestion code block like 👇 (or press the `[±]` button in GitHub's Markdown editor) it'll be clear that you're making a suggestion.

```markdown
    ```suggestion
    While they can be useful, we don't generally use domestic key constraints in Grafana, for historical and
    ```
```

turns into

```suggestion
While they can be useful, we don't generally use domestic key constraints in Grafana, for historical and
```

Reply:
Oh, this is brilliant. Thank you!